SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/visitors/ToRustInnerVisitor.java

```diff
@@ -60,7 +58,7 @@ public boolean preorder(DBSPLiteral literal) {
             this.builder.append(literal.noneString());
             return false;
         }
-        return true;
+        return true; // intentionally true
```

Comment:
Specify the intention in the comment as well?

Reply:
Will do in a subsequent PR, the CI takes too long.